src/codelabs/dart-cheatsheet.md

```diff
@@ -88,27 +88,25 @@ reference them inside single quotes, with a space in between.
 
 ## Nullable variables
 
-Dart 2.12 introduced sound null safety,
-meaning that (when you [enable null safety](/null-safety#enable-null-safety))
-values can’t be null unless you say they can be.
-In other words, types are non-nullable by default.
+Dart enforces sound null safety.
+This means values can’t be null unless you say they can be.
+In other words, types default to non-nullable.
 
-For example, consider the following code,
-which is **invalid** because (with null safety)
-a variable of type `int` can't have the value `null`:
+For example, consider the following code.
+With null safety, this code returns an error.
+A variable of type `int` can't have the value `null`:
 
 <?code-excerpt "misc/bin/cheatsheet/nullable.dart (invalid-null)" replace="/null;/[!null!];/g"?>
 ```dart
-int a = [!null!]; // INVALID in null-safe Dart.
+int a = [!null!]; // INVALID
```

Comment:
This is a code excerpt so you will need it update it in its original file: https://github.com/dart-lang/site-www/blob/main/examples/misc/bin/cheatsheet/nullable.dart#L7